lib/CreepPlayers/Behaviors/MoveWithFriendDumb.js

```diff
@@ -0,0 +1,34 @@
+
+var MoveUtils = require('../../Utils/MoveUtils');
+var Base = require('./Base');
+var UnitType = require('../../UnitType');
+var GameConstants = require('../../GameConstants');
+
+/**
+ *
+ * @param creepController @type CreepController
+ * @constructor
+ */
+function MoveWithFriendDumb(creepController) {
+    this.cc = creepController;
+}
+
+MoveWithFriendDumb.prototype = Object.create(Base.prototype);
+
+MoveWithFriendDumb.prototype.act = function() {
+    var toPlayer = ourLoc.directionTo(playerLoc);
+	var friends = this.cc.senseNearbyUnitsFromTeam(this.unitInfo().team);
```

Comment:
this.unitInfo() shouldn't work (or if it does, security issues!) this.cc.getUnitInfo() should be fine